keps/0032-community-repository-management.md

```diff
@@ -0,0 +1,96 @@
+---
+kep-number: 32
+title: Community Repository Management
+short-desc: Details on how to add operator to the community repository
+authors:
+  - "@nfnt"
+owners:
+  - "@nfnt"
+creation-date: 2020-07-02
+last-updated: 2020-07-02
+status: provisional
+see-also:
+  - KEP-10
+  - KEP-15
+---
+
+# Community Repository Management
+
+## Table of Contents
+
+- [Community Repository Management](#community-repository-management)
+  - [Table of Contents](#table-of-contents)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Implementation History](#implementation-history)
+
+## Summary
+
+By default, every KUDO deployment installs operator packages from the community repository. As we encourage the KUDO community to add new operator packages to this repository, this KEP defines
+
+- The workflow to add new operator packages to the community repository
+- Rules and means to describe the maturity of operator packages in the community repository
+
+## Motivation
+
+Currently, every operator package in the community repository is provided from the `kudobuilder/operators` Git repository. This Git repository contains package definitions, documentation and sometimes tests for each operator. While it is convenient, to have this all in a single repository, this is challenging for larger or third-party operators:
+
+- Large operators usually provide own Docker images and/or additional dependencies. These have to be build, tested and deployed. The tools for this can't be provided by `kudobuilder/operators`.
```

Comment:
```suggestion
- Large operators usually provide their own Docker images and/or additional dependencies. These have to be built, tested and deployed. The tools for this can't be provided by `kudobuilder/operators`.
```